server/boot/user.js

```diff
@@ -608,7 +608,14 @@ module.exports = function(app) {
     return User.findById(req.accessToken.userId, function(err, user) {
       if (err) { return next(err); }
       return user.updateAttribute('password', password, function(err) {
-        if (err) { return next(err); }
+        if (err) {
+          debug(err);
+          req.flash('error', {
+            msg: err.message ||
```

Comment:
See comment above and as I mentioned earlier

> Note: I have although left the error message display to a raw message returned from loopback, this is done intentionally because the entire reset logic is going away in any case with passwordless.

This entire patch is temporary for now.

I am okay though to not intercept the message, but that would not allow us to show the actual validation from LoopBack.